### ISSUE TITLE
Map.! error when incorporating `mdgriffith/elm-style-animation` (another SSCCE)

### ISSUE THREAD

**Issue by clozach** [elm/compiler#1877]
_Does your title concisely summarize the problem?_ ✔️
_Did you include a minimal, reproducable example? Is it an SSCCE?_ ✔️

_What OS and browser are you using? What versions?_

    macOS 10.13.6 (17G4015)
    Chrome 71.0.3578.98 (Official Build) (64-bit)

_What version of Elm are you using?_
   
    0.19

I've put the SSCCE here: https://github.com/clozach/sscce-for-elm-compiler-issue-1851.

Steps to reproduce are in the README.

**Comment by kyasu1** [comment]
The recent Parcel is compiling with `--debug` enabled, that is causing your problem. You can downgrade to version 1.10.3 which did not have this option. Please refer https://github.com/parcel-bundler/parcel/pull/2626#issuecomment-462212763.

**Comment by domenkozar** [comment]
There is also proposed fix in https://github.com/elm/compiler/pull/1850

**Comment by evancz** [comment]
I believe this will be fixed by the changes described in this comment: https://github.com/elm/compiler/pull/1850#issuecomment-509327575

I would like to confirm that it works here, but I would not consider this an SSCCE. I would expect to see a single `Main.elm` file and a single `elm.json` file all inline in the issue. It is not practical to try reproducing things with parcel. If it is calling `elm` then that would be part of the SSCCE.

I think it makes sense to close since (1) I think this will be fixed when 0.19.1 and (2) I am not clear on how to reproduce this exactly and confirm myself.

**Comment by clozach** [comment]
Boy, it's been a long time since I clicked on my github notifications. 😳 Apologies for the extremely delayed response!

### TL;DR

My first attempt was created in ignorance, but now I've updated my SSCCE to be truly minimal with far clearer steps to reproduce. No action is needed by anyone here 

### LV;FFTS*

_*Long version; Feel free to skip_

`Viir` wrote:

> Why does this not include the --debug flag mentioned in the referenced issue?

By the time I got around to attempting the SSCCE, I'd been wrestling with the error for hours. In my mentally-depleted state, I didn't notice that the `--debug` flag was relevant.

> Why do you depend on Parcel in your example?

The answer's partly because I didn't understand what `kyasu1` helpfully pointed out: Parcel compiles with `--debug` by default. I first encountered the issue when trying to start a new project using Parcel. Not understanding the importance of the flag in repro'ing the error, I reported it as I was able to reproduce it, namely, with Parcel. 🙄

Wanting to have something cleaner to test against once 0.19.1 comes out, I simplified the code in my [sscce repo](https://github.com/clozach/sscce-for-elm-compiler-issue-1851) to just `elm.json` + `Main.elm`. I also boiled the README down so it should now be _easy_ to follow. 🤓
